Google.Solutions.Compute/Auth/GcloudAccount.cs

```diff
@@ -0,0 +1,106 @@
+﻿//
+// Copyright 2019 Google LLC
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+using Google.Apis.Auth.OAuth2;
+using System;
+using System.IO;
+using System.Text;
+
+namespace Google.Solutions.Compute.Auth
+{
+    /// <summary>
+    /// Allows reading account information and credentials of the active
+    /// GCloud account of the current user's profile.
+    /// </summary>
+    public class GcloudAccount
+    {
+        private readonly string account;
+
+        public GcloudAccount(string account)
+        {
+            this.account = account;
+        }
+
+        internal static string FolderPath
+        {
+            get
+            {
+                return Path.Combine(
+                    Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData),
+                    "gcloud");
+            }
+        }
+
+        public static GcloudAccount ActiveAccount
+        {
+            get
+            {
+                var configPath = Path.Combine(FolderPath, "configurations", "config_default");
```

Comment:
do you need to have initialized gcloud atleast once for the config path to exist (i.,e is mere installation of the gcloud sdk create this)?  Later on, gcloud core/account wont' exist unless the person did a gcloud auth login